src/mod/endpoints/mod_aes67/mod_aes67.c

```diff
@@ -3340,6 +3353,7 @@ SWITCH_STANDARD_API(aes_cmd)
       if (STREAM_READER_TRYLOCK(astream)) {
           drop_output_buffers(FALSE, astream->stream);
           astream->txflow = TRUE;
+          astream->stream->txdrop = FALSE;
```

Comment:
Is the `txflow` boolean now redundant? Should we just use the `txdrop` flag everywhere instead?

Reply:
`txflow` is needed to read from the config and then used it to set the valve's `drop` property during the pipeline init. 
I tried to keep just the stream->drop throughout but the stream is initialized only after the config is read.

Reply:
Might be good to document that for our future selves :)